content/applications/websites/ecommerce/managing_products/price_management.rst

```diff
@@ -0,0 +1,204 @@
+================
+Price management
+================
+
+Odoo offers multiple options to select the price displayed on your website, as well as
+condition-specific prices based on selected criteria.
+
+Displayed price
+===============
+
+Choosing the displayed tax usually depends on a country's regulations or the type of customers
+**(B2B vs B2C)**. To select the type of price displayed, go to :menuselection:`Website app -->
+Configuration --> Settings`, scroll-down to the :guilabel:`Shop - Products` category, and select
+:guilabel:`Tax Excluded` or :guilabel:`Tax Included`.
```

Comment:
```suggestion
================
Price management
================

Odoo offers multiple options to select the price displayed on your website, as well as
condition-specific prices based on selected criteria.

Displayed price
============

Choosing the displayed tax usually depends on a country's regulations or the type of customers
**(B2B vs B2C)**. To select the type of price displayed, go to :menuselection:`Website app -->
Configuration --> Settings`, scroll-down to the :guilabel:`Shop - Products` category, and select
:guilabel:`Tax Excluded` or :guilabel:`Tax Included`.
```

Reply:
I think there might be something wrong between your setup and my setup, as H-level should always be the same length as the title themselves. Your suggestion would not be the same length anymore :) 
![jaae](https://user-images.githubusercontent.com/104567387/205001015-c749d15a-7238-457d-8e99-a8a5380ff501.png)